cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,70 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SetingsPageObject from '../support/pages/settings.pageObject'
+import SignInPageObject from '../support/pages/signIn.pageObject'
+
+const signInPage = new SignInPageObject()
+const settingsPage = new SetingsPageObject()
 describe('Settings page', () => {
+  let user
   before(() => {
-
-  });
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser
+    })
+  })
 
   beforeEach(() => {
+    cy.task('db:clear')
 
-  });
+    signInPage.visit()
+    cy.register(user.email, user.username, user.password)
 
+    signInPage.typeEmail(user.email)
+    signInPage.typePassword(user.password)
+    signInPage.clickSignInBtn()
+    settingsPage.visit()
+  })
   it('should provide an ability to update username', () => {
+    settingsPage.userNameField.type('new')
```

Comment:
```suggestion
    settingsPage.userNameField.type('new')
```
this is not page object method. Wrap this and similar methods inside page object methods